test/run_tests.sh

```diff
@@ -26,10 +30,24 @@ if ! docker inspect "$builder_image"; then
 fi
 
 if docker build -f test/Dockerfile -t kudo-test .; then
-    if docker run -e INTEGRATION_OUTPUT_JUNIT --net=host -it -m 4g -v /var/run/docker.sock:/var/run/docker.sock -v "$(pwd)"/reports:/go/src/github.com/kudobuilder/kudo/reports --rm kudo-test make "$TARGET"; then
+    docker run -e INTEGRATION_OUTPUT_JUNIT --net=host -it -m 4g \
+        --name kudo-e2e-test \
```

Comment:
We may run into issues here if for some reason the name is already taken. Might be better to use a generated name?

Reply:
Agreed, let's see how to do this in a shell script.